src/components/header/Alarm/index.tsx

```diff
@@ -27,9 +25,9 @@ const Alarm = ({ isAlarmExisted, isActivated, onClick, alarmRef }: AlarmProps) =
       <div className={cx('dot', 'dot-left', { 'dot-activated': isActivated })}></div>
       <div className={cx('frame-inner', { 'frame-inner-activated': isActivated })}>
         {isAlarmExisted ? (
-          <Image src={full.url} alt={full.alt} width={20} height={20} />
+          <img src={full.url} alt={full.alt} width={20} height={20} />
         ) : (
```

Comment:
상수로 선언하고 url, alt만 내려주면 어떨까요?

```suggestion
    const {url, alt} = isAlarmExisted ? full : empty
    // 생략...
    <img src={url} alt={alt} width={20} height={20} />
```